src/openfl/display/Bitmap.hx

```diff
@@ -301,6 +336,7 @@ class Bitmap extends DisplayObject implements IShaderDrawable {
 		smoothing = false;
 		
 		__setRenderDirty ();
+		__batchQuadDirty = true;
```

Comment:
Is there any other cases besides changing `bitmapData` or transform where we need to invalidate the quad? I guess not...